repowiz-infrastructure/src/main/groovy/life/qbic/repowiz/prepare/projectSearch/OpenBisProjectSearcher.groovy

```diff
@@ -128,39 +151,51 @@ class OpenBisProjectSearcher extends ProjectSearcher implements ProjectSearchInp
             HashMap sampleProperties = collectProperties(sample)
             sampleValidation(sampleProperties)
 
-            repoWizSamples << new RepoWizSample("Sample "+counter, sampleProperties)
-            counter ++
+            repoWizSamples << new RepoWizSample("Sample " + counter, sampleProperties)
```

Comment:
```suggestion
            repoWizSamples =new RepoWizSample("Sample " + counter, sampleProperties)
```
? I would try to avoid special groovy operators since they decrease readability in my opinion. I am not sure if `=` is correct here.

Reply:
Shift operators have quite the different meaning in Java and Groovy. Groovy 'verschlimmbessert' again.

Reply:
It means that it adds the sample to a list, replacing it with '=' would not work here. You would need to replace it with .add() or is it .put()? Nevertheless, I would keep it as it works and I think it is easy to read. Just imaging the arrows show where the data is shoved into.